src/js/components/Value.js

```diff
@@ -59,7 +59,7 @@ export default class Value extends Component {
       contentNode = (
         <div>
           <span className={`${CLASS_ROOT}__value`}>
-            {value}
+            {parseFloat(value.toFixed(2))}
```

Comment:
can the caller do this? I'm wondering if there are cases where you would want 4 decimal places instead. Can we just assume that whoever is passing `value` will format it properly?

Reply:
Agreed. For my application the customer has the ability to configure the number of digits after the decimal.

Reply:
Even if I pass the values such as:
```
  const storageData = [
    {
      value: parseFloat(storageUsed.size.toFixed(2)),
      units: storageUsed.units,
      label: 'In use', colorIndex: 'graph-2'},
    {
      value: parseFloat(storageAvailable.size.toFixed(2)),
      units: storageAvailable.units,
      label: 'Available', colorIndex: 'graph-1'}
  ];
```

The decimal error will still happen because the sum isn't formatted.

>1.2+0.6
<-1.7999999999999998
>1.25+0.68
<-1.9300000000000002

Reply:
We only have control on the input parameters of the AnnotedMeter, not the sum of the series value.
```
a = 1.25
_1.25_
b = 0.68
_0.68_
parseFloat(a.toFixed(2)) + parseFloat(b.toFixed(2)) 
_1.9300000000000002_
```

Reply:
I think this is an issue with `AnnotatedMeter` then. I've seen enough variety in how `Value` is used to not want to make this sort of change in Value.

Reply:
![image](https://user-images.githubusercontent.com/1621145/34648647-c6758960-f3b6-11e7-9f01-04aa69c87698.png)

Agreed, @ericsoderberghp see the above screenshot.